flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -229,17 +229,26 @@ web.port: 8081
 
 # Directory to upload completed jobs to. Add this directory to the list of
 # monitored directories of the HistoryServer as well (see below).
-#jobmanager.archive.fs.dir: hdfs:///completed-jobs/
+# jobmanager.archive.fs.dir: hdfs:///completed-jobs/
 
 # The address under which the web-based HistoryServer listens.
-#historyserver.web.address: 0.0.0.0
+# historyserver.web.address: 0.0.0.0
 
 # The port under which the web-based HistoryServer listens.
-#historyserver.web.port: 8082
+# historyserver.web.port: 8082
 
 # Comma separated list of directories to monitor for completed jobs.
-#historyserver.archive.fs.dir: hdfs:///completed-jobs/
+# historyserver.archive.fs.dir: hdfs:///completed-jobs/
 
 # Interval in milliseconds for refreshing the monitored directories.
-#historyserver.archive.fs.refresh-interval: 10000
+# historyserver.archive.fs.refresh-interval: 10000
+
+#==============================================================================
+# SQL Client
+#==============================================================================
+
+# The SQL Client CLI can be started via bin/sql-client.sh embedded
+
+# The heap size for the SQL Client CLI JVM
+# sqlclient.cli.heap.mb: 1024
```

Comment:
Regarding the question whether to add this property of the `flink-conf.yaml` or a CLI client specific file, are there other (existing) properties in the `flink-conf.yaml` file that are used for the CLI client, besides the connection info to the cluster?

Reply:
If the user wants to specify global JVM options for all Flink scripts, this also happens in the `flink-conf.yaml`.

Reply:
OK. But there's another discussion thread whether it is useful to share properties across JM/TM JVMs and client processes.